src/main/scala/org/constellation/storage/SnapshotService.scala

```diff
@@ -3,9 +3,11 @@ package org.constellation.storage
 import java.io.FileOutputStream
 import java.nio.file.Path
 
+import better.files.File
+import cats.Parallel
 import cats.data.EitherT
 import cats.effect.concurrent.Ref
-import cats.effect.{Concurrent, ContextShift, IO, LiftIO, Resource, Sync}
+import cats.effect.{Concurrent, ContextShift, LiftIO, Resource, Sync, _}
```

Comment:
if you're using `import _`, then you can remove `{Concurrent, ...}` and keep only `import cats.effect._`